EGaDS! Open Project Game/Assets/Scripts/AbilitySystem/PlayerController.cs

```diff
@@ -22,6 +22,15 @@ public enum Facings { LEFT = -1, RIGHT = 1 };
     [SerializeField] private float _wallDetectionWidth = .8f;
     [SerializeField] private float _wallDetectionHeight = .7f;
 
+    [Header("Dash Parameters")]
+    [SerializeField] private float _dashingPower = 24f; // Determines speed of the dash
+    [SerializeField] private float _dashingTime = 0.2f; // Determines how long the dash stays in effect
+    [SerializeField] private float _dashInputForgiveness = 0.1f; // Determines how late a direction can be pressed and still be processed after dash is pressed
+    [SerializeField] private TrailRenderer _trailRend; // Reference to the Trail Renderer
+    [HideInInspector] private bool _canDash = true; // Indicates if another dash is permitted
+    [HideInInspector] private bool _isDashing; // Indicates if player is currently dashing
```

Comment:
Nit: hid in inspector not necessary for private fields
```suggestion
    private bool _canDash = true; // Indicates if another dash is permitted
    private bool _isDashing; // Indicates if player is currently dashing
```